content/SCALE/GettingStarted/VMandAppConfigSCALE.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Configuring Virtualization and Apps in TrueNAS SCALE"
+weight: 50
+tags:
+- scalevm
+- scaleapps
+- scalenetwork
+- scalevlan
+- scalelagg
+- scaledatashare
+---
+
+{{< toc >}}
+
+Configuring TrueNAS SCALE to work with virtualized features, such as VMs and Apps, is a part of the setup process that when optimized takes advantage of the network storage capabilities that SCALE offers. 
+
+## Before You Begin
+
+This article assumes that you have the latest release version of TrueNAS SCALE installed on your system. The following steps comprise a list of configuration prerequisites that you have completed and are familiar with before beginning VM and App installation.
+
+### Network Prerequisites
+
+The primary network interface is configured as part of the SCALE installation process. Go to **Network > Global Configuration** screen in the TrueNAS web UI to determine that the default gateway, host name, domain and DNS name servers have been configured correctly. See [Console Setup Menu Configuration]({{< relref "ConsoleSetupMenuScale.md" >}}) for more information on network settings.
+
+### Optional: Network VLAN 
+
+You have the option of configuring a virtual LAN (VLAN) to route traffic for your VMs. VLAN benefits include the reduction of broadcast traffic and the ability to group resources in different physical locations into a broadcast domain. VLANs virtually segment a network. Different VLANs can communicate with each other using layer 3 devices. See [Setting Up a Network VLAN]({{< relref "SettingUpVLAN.md" >}}) for more information on creating virtual LANs (VLAN).
+
+### Storage Prerequisites
+
+Storage pool creation is part of the initial process of setting up storage for SCALE. A TrueNAS dataset is a file system within a data storage pool. See [Setting Up Storage]({{< relref "SetUpStorageScale.md" >}}) to review storage pool creation and [Adding and Managing Datasets]({{< relref "DatasetsSCALE.md" >}}) for information on dataset configuration. 
+
+### Set Up Shares
+
+Once the pool and datasets have been created, set up shares to enable data access. Three types of data sharing methods are discussed in 
+[Setting Up Data Sharing]({{< relref "SetUpSharing.md" >}}). More specific coverage of each share should be investigated based on your individual use case. 
+
+A good introduction as to how TrueNAS SCALE handles SMB shares is located at [SMB Shares Screens]({{< relref "SMBSharesScreens.md" >}}) and [Setting Up SMB Home Shares]({{< relref "AddSMBHomeShare.md" >}}). For a better understanding of how to configure Active Directory, see [Active Directory Screen]({{< relref "scale/scaleuireference/credentials/directoryservices/ActiveDirectory.md" >}}). Required Kerberos information is outlined here, and also in this article: [Configuring Kerberos]({{< relref "ConfigKerberosSCALE.md" >}}). Configuring LDAP best practices can be found at: [Configuring LDAP]({{< relref "ConfigLDAPSCALE.md" >}}).
```

Comment:
Better organization of this section, maybe try
```suggestion
[SMB Shares Screens]({{< relref "SMBSharesScreens.md" >}}) and [Setting Up SMB Home Shares]({{< relref "AddSMBHomeShare.md" >}}) provide a good introduction as to how TrueNAS SCALE handles SMB shares. See [Active Directory Screen]({{< relref "scale/scaleuireference/credentials/directoryservices/ActiveDirectory.md" >}}) for a better understanding of how to configure Active Directory, and [Configuring Kerberos]({{< relref "ConfigKerberosSCALE.md" >}}) for an outline of required Kerberos information. For LDAP best practices see [Configuring LDAP]({{< relref "ConfigLDAPSCALE.md" >}}).
```
Also, it is a bit of a non-sequitur to lump shares and directory services together in this paragraph and before you mention NFS shares. They are not the same thing so this is a bit confusing. Make the directory services information a separate paragraph or a new heading such as 
### Directory Services Configuration

Reply:
Shares content has been updated. Directory services are now under a separate header.